index.html

```diff
@@ -80,7 +80,109 @@ <h1 class="mn__title">
             /></a>
           </li>
         </ul>
+        <div class="arrow_down">
+        <img
+          src="https://img.icons8.com/ios/45/172B4D/circled-chevron-down.png"
+          alt="circled arrow-down icon"
+        />
+      </div>
       </section>
+      <section class="works__section">
```

Comment:
- [ ] You did a good job using the `<section>` tag to separate 'Works' :+1: , but as a [project requirement](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/2_mobile.md) you should lay out the elements inside of it using CSS Grid and Flexbox combined. Use CSS Grid in the main container that contains all of the cards and use Flexbox to lay out an individual card :+1: :100: .